app/test_app/functional/api/test_transfer_card_api.py

```diff
@@ -135,7 +136,7 @@ def bind_user(id, serial_number):
     bind_user(user9.id, 888888)
 
     # Make sure the distance filters are working!
-    distances = [(None, 9), (0, 9), (0.001, 7), (2, 7), (10, 8), (100, 9), (100000, 10)]
```

Comment:
actually a bug in the previous code which was picking up an location with none or something (can't remember exactly what, been a while since i wrote this)